clients/src/main/java/org/apache/kafka/common/log/remote/metadata/storage/ConsumerTask.java

```diff
@@ -133,7 +149,7 @@ public void run() {
                                 break;
                             } catch (Exception e) {
                                 // ignore exception
-                                log.info("#### Error encountered in fetching end offsets");
+                                log.info("Error encountered in fetching end offsets");
                             }
                         }
                         log.info("Fetched end offsets to consumer task [{}]", endOffsets);
```

Comment:
This happens very frequently. I think it's better to make it a debug message rather than info message